content/klc/S2.2.adoc

```diff
@@ -10,10 +10,10 @@ Examples:
 * Packaging information (e.g. reel, tray, tape)
 * RoHS / PbFree information
 
-To capture every possible combination of part variation requires a large number of symbol aliases - this is to be avoided.
+To capture every possible combination of part variation would require a large number of symbol aliases - this is to be avoided.
 
-Where an MPN has options for such non-functional variations, these portions of the MPN should be replaced with a wildcard character (`x`).
+Where a MPN has options for such non-functional variations, these portions of the MPN should be replaced with a wildcard character (`x`).
```

Comment:
give examples

Reply:
This was this way before my changes and i think it should be fixed in a separate PR. (I only fixed a spelling error here.)
Otherwise this PR will never be finished.